### DISCUSSION TITLE
Combobox implementation of Autocomplete Country Dropdown

### DISCUSSION THREAD

**Discussion by andelopes** [twilio-labs/paste#1503] [Office hours] [answered]
Hi all,

Wanted to discuss my implementation of a Country Dropdown using the Paste Combobox.

I've implemented functionality that resembles Material UI's Autocomplete and wanted feedback on the interaction and implementation. This is related to the "Buy a Number" improvements in Console.
https://github.com/twilio-labs/paste/discussions/1420#discussioncomment-701304

Example:
https://code.hq.twilio.com/pages/numbers/numbers-toolbox/?path=/story/phone-numbers-countries-countrydropdown--dropdown-with-default-countries

Implementation:
https://code.hq.twilio.com/numbers/numbers-toolbox/blob/master/packages/frontend/src/domains/PhoneNumbers/Countries/CountryDropdown/component.tsx#L89-L128

**Comment by SiTaggart** [comment, marked as answer]
Hi

Thanks for coming to Office Hours this week. Here is a link to the recording https://www.loom.com/share/17d88430c6cb40fead29a5dc4348ef91, the passcode for which can be found in [Slack](https://twilio.slack.com/archives/CBEQHHDLP/p1622245473021100
